trestlebot/entrypoints/init.py

```diff
@@ -204,7 +213,7 @@ def run(self, args: argparse.Namespace) -> None:
             traceback_str = traceback.format_exc()
             exit_code = handle_exception(e, traceback_str)
 
-        logger.info(f"Initialized trestlebot project successfully in {root}")
+        logger.warning(f"Initialized trestlebot project successfully in {root}")
```

Comment:
Trying to understand the change to `warning` here and the message seems oriented for the `info` level. Can you expand on it?

Reply:
I believe this is an artifact from some local testing.  I agree this should be at the `INFO` level, I will revert it.